lighthouse-core/audits/manifest-display.js

```diff
@@ -53,9 +53,14 @@ class ManifestDisplay extends Audit {
     return ManifestDisplay.generateAuditResult({
       rawValue: hasRecommendedValue,
       displayValue,
-      debugString: 'Manifest display property should be set.'
+      debugString: this.createDebugString(hasRecommendedValue)
     });
   }
+
+  static createDebugString(hasRecValue) {
+    const debugString = 'Manifest display property should be set.';
+    return hasRecValue ? '' : debugString;
```

Comment:
definitely agree that `debugString` should be undefined if no error. Doing `return hasRecValue ? undefined : debugString;` here would be sufficient